integration/__tests__/lerna-publish-conventional-fixed-prerelease.spec.ts

```diff
@@ -438,3 +438,285 @@ describe(`lerna publish --conventional-prerelease/graduate fixed w/ changelog`,
+
+      Changes:
+       - package-1: 2.0.0-alpha.0 => 2.0.0
+       - package-2: 2.0.0-alpha.0 => 2.0.0
+       - package-3: 2.0.0-alpha.0 => 2.0.0
+       - package-4: 1.0.0 => 2.0.0
+       - package-5: 2.0.0-alpha.0 => 2.0.0 (private)
+
+      Successfully published:
+       - package-1@2.0.0
+       - package-2@2.0.0
+       - package-3@2.0.0
+       - package-4@2.0.0
+    `);
+  });
+
+  test(`generate accurate changelog`, async () => {
+    // ensure changelog header is not duplicated
+    const args = ["publish", "--conventional-commits", "--changelog-skip-unstable", "--yes"];
+    await commitChangeToPackage(cwd, "package-2", "fix(package-2): And another thing", { thing: true });
+    await cliRunner(cwd, env)(...args);
+
+    const changelogFilePaths = await globby(["**/CHANGELOG.md"], {
+      cwd,
+      absolute: true,
+      followSymbolicLinks: false,
+    });
+    const [rootChangelog, pkg1Changelog, pkg2Changelog, pkg3Changelog] =
+      await Promise.all(changelogFilePaths.sort().map((fp) => fs.readFile(fp, "utf8")));
+
+    /**
+     * ./CHANGELOG.md
+     */
+    expect(rootChangelog).toMatchInlineSnapshot(`
+      # Change Log
+
+      All notable changes to this project will be documented in this file.
+      See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+      ## [2.0.1](/compare/v2.0.0...v2.0.1) (YYYY-MM-DD)
+
+
+      ### Bug Fixes
+
+      * **package-2:** And another thing ([SHA](COMMIT_URL))
+
+
+
+
+
+      # [2.0.0](/compare/v1.0.0...v2.0.0) (YYYY-MM-DD)
+
+
+      ### Features
+
+      * **package-1:** Add baz ([SHA](COMMIT_URL))
+
+
+
+      # 2.0.0-alpha.0 (YYYY-MM-DD)
+
+
+      ### Bug Fixes
+
+      * **package-2:** Fix bar ([SHA](COMMIT_URL))
+
+
+      ### Features
+
+      * **package-1:** Add foo ([SHA](COMMIT_URL))
+      * **package-3:** Add baz feature ([SHA](COMMIT_URL))
+
+
+      ### BREAKING CHANGES
+
+      * **package-3:** yup
+
+
+
+
+
+      # [2.0.0-alpha.0](/compare/v1.0.0...v2.0.0-alpha.0) (YYYY-MM-DD)
+
+
+      ### Bug Fixes
+
+      * **package-2:** Fix bar ([SHA](COMMIT_URL))
+
+
+      ### Features
+
+      * **package-1:** Add foo ([SHA](COMMIT_URL))
+      * **package-3:** Add baz feature ([SHA](COMMIT_URL))
+
+
+      ### BREAKING CHANGES
+
+      * **package-3:** yup
+
+    `);
+
+    expect(pkg1Changelog).toMatchInlineSnapshot(`
+      # Change Log
+
+      All notable changes to this project will be documented in this file.
+      See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+      # [2.0.0](/compare/v1.0.0...v2.0.0) (YYYY-MM-DD)
+
+
+      ### Features
+
+      * **package-1:** Add baz ([SHA](COMMIT_URL))
+
+
+
+      # 2.0.0-alpha.0 (YYYY-MM-DD)
+
+
+      ### Features
+
+      * **package-1:** Add foo ([SHA](COMMIT_URL))
+
+
+
+
+
+      # [2.0.0-alpha.0](/compare/v1.0.0...v2.0.0-alpha.0) (YYYY-MM-DD)
+
+
+      ### Features
+
+      * **package-1:** Add foo ([SHA](COMMIT_URL))
+
```

Comment:
2.0.0-alpha.0 appears twice in this markdown file. This shouldn't happen, as users looking at version 2.0.0 won't care which specific prerelease version the feature was added in. Instead, we should get the changes from all the prereleases aggregated together with the changes from 2.0.0 itself, so it would look like this:
```
# [2.0.0](/compare/v1.0.0...v2.0.0) (YYYY-MM-DD)


### Features

* **package-1:** Add baz ([SHA](COMMIT_URL))
* **package-1:** Add foo ([SHA](COMMIT_URL))


# [2.0.0-alpha.0](/compare/v1.0.0...v2.0.0-alpha.0) (YYYY-MM-DD)


### Features

* **package-1:** Add foo ([SHA](COMMIT_URL))
```

Reply:
@fahslaj I do agree that this is the expected markup. 

https://github.com/conventional-changelog/conventional-changelog/issues/675 suggests this is an issue with `conventional-changelog`. However, tests in this PR with independent versioning seem to be correct and the prerelease changes are properly aggregated [see here](https://github.com/lerna/lerna/pull/3882/files/245a739118716bcf38849acf6e737c3ff78392fe#diff-dc2f58b3f119430f0a847687e18e51fb3a62e6dd131e0aea813455fcaed7cb18). Any suggestions ?